packages/foreman/rubygem-gettext/rubygem-gettext.spec

```diff
@@ -1,181 +1,108 @@
+BuildRequires: %{?scl_prefix_ruby}ruby 
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel 
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
 
 %description
-Ruby-GetText-Package is a GNU GetText-like program for Ruby.
-The catalog file(po-file) is same format with GNU GetText.
-So you can use GNU GetText tools for maintaining.
+Gettext is a GNU gettext-like program for Ruby.
+The catalog file(po-file) is same format with GNU gettext.
+So you can use GNU gettext tools for maintaining.
 
-This package provides gem for Ruby-Gettext-Package.
 
-%package	doc
-Summary:	Documentation for %{pkg_name}
-Group:		Documentation
-Requires:	%{?scl_prefix}%{pkg_name} = %{version}-%{release}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}-doc}
-
-%description	doc
-This package contains documentation for %{pkg_name}.
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
 
+%description doc
+Documentation for %{pkg_name}.
 
 %prep
-%setup -n %{pkg_name}-%{version} -q -c -T
-
-TOPDIR=$(pwd)
-mkdir tmpunpackdir
-pushd tmpunpackdir
-
-%{?scl:scl enable %{scl} "}
+%{?scl:scl enable %{scl} - << \EOF}
 gem unpack %{SOURCE0}
-%{?scl:"}
-cd %{gem_name}-%{version}
+%{?scl:EOF}
 
-#Patches, etc
+%setup -q -D -T -n  %{gem_name}-%{version}
 
-%{?scl:scl enable %{scl} "}
-gem specification -l --ruby %{SOURCE0}
-%{?scl:"} > %{gem_name}.gemspec
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
 
-%{?scl:scl enable %{scl} "}
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
 gem build %{gem_name}.gemspec
-%{?scl:"}
-mv %{gem_name}-%{version}.gem $TOPDIR
-
-popd
-rm -rf tmpunpackdir
+%{?scl:EOF}
 
-%build
-%{?scl:scl enable %{scl} - <<EOF}
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
 %gem_install
 %{?scl:EOF}
 
-#%%{__rm} -f .%{gem_instdir}/Rakefile
-%{__rm} -f .%{gem_instdir}/%{gem_name}.gemspec
-%{__rm} -rf .%{gem_instdir}/po/
-%{__chmod} 0755 .%{gem_instdir}/bin/*
-%{__chmod} 0644 .%{gem_dir}/cache/*.gem
-find .%{gem_instdir}/ -name \*.po | xargs %{__chmod} 0644
-
-# Cleanups for rpmlint
-find .%{gem_instdir}/lib/ -name \*.rb | while read f
-do
-	%{__sed} -i -e '/^#!/d' $f
-done
-
 %install
-%{__mkdir_p} %{buildroot}{%{gem_dir},%{_bindir}}
-
-%{__cp} -a .%{_bindir}/* %{buildroot}/%{_bindir}/
-%{__cp} -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
-find %{buildroot}%{gem_dir} -name \*.rb.patch\* -delete
-
-
-# For --short-circult
-%{__rm} -f *.lang
-
-# modify find-lang.sh to deal with gettext .mo files under
-# %%{gem_instdir}/locale (required on EL6)
-sed -e 's|/share/locale/|/locale/|' \
-	/usr/lib/rpm/find-lang.sh \
-	> find-lang-modified.sh
```

Comment:
I don't feel like any of the stuff in here should be used in production so it ought to be OK to omit it. @ekohl do you know?

Reply:
Looks like it's commented out in https://koji.fedoraproject.org/koji/buildinfo?buildID=1432155 at least. However there's various changes here that actually trim things down, like removing the `po` directory.

Reply:
I don't think we need that either in production? This appears as just a runtime dependency of the rails i18n gem in our deployment.